### DISCUSSION TITLE
Referencing Diataxis

### DISCUSSION THREAD

**Discussion by drvinceknight** [evildmp/diataxis-documentation-framework#34] [Q&A] [answered]
Any thoughts/preference on how to reference/cite the framework?

Currently in a project I'm documenting I have gone with:

```
Daniele Procida. Diátaxis: A Systematic Framework For Technical Documentation Authoring. https://diataxis.fr
```

![CleanShot 2021-05-05 at 15 12 00](https://user-images.githubusercontent.com/2131546/117154950-45b2db00-adb4-11eb-8483-e6340e9ceb01.png)

One possibility could be to "plug" the repo in to a service like [Zenodo](https://zenodo.org) and then make releases. This would automatically generate a DOI that would be standardly citable but would also point at archived versions.

Follow up question: depending on the answer to the above would a PR adding a "How to reference the framework" section be helpful?

**Comment by evildmp** [comment, marked as answer]
Sorry, I forgot about Discussions...

The way you've suggested looks absolutely fine. However if there is a useful reusable standard for citation, even better.

I would suggest that the https://diataxis.fr/development/ page would be a good place to add a "How to reference the framework" section.

Thanks for the suggestion.

**Reply by cmungall** [reply]
You might also want to add a citation.cff file -- see https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files

It is a fantastic and incredibly useful framework, many of us would love to make sure we are giving credit appropriately!

**Reply by evildmp** [reply]
Done, thanks for the suggestion.
